powerdns/migrations/0031_auto_20161028_0324.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('powerdns', '0030_auto_20161026_0944'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='domainrequest',
```

Comment:
afaik we don't support domain requests anymore and it's going to be deleted soon?